www/src/css/_animations.scss

```diff
@@ -0,0 +1,8 @@
+@use './var' as *;
+
+@mixin subtle-border-animation {
+  &:hover {
+    border-color: transparent; /* remove the border's colour */
+    box-shadow: 0 0 0 2px rgba(46, 95, 132, 0.69); /* emulate the border */
```

Comment:
Should this have a `transition` to animate or did we want this to be an instant effect?